causely/assets/dashboards/causely_dashboard.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
Sending you a dashboard template via email that you can use and fill out so it aligns with out best practices. Please make sure you fill out what is included in the template.

Reply:
thanks, I tried to follow the template and reworked the previous dashboard components, I hope it is ok